### DISCUSSION TITLE
Magick.Net TIFF to PDF causing black background instead of white

### DISCUSSION THREAD

**Discussion by DougWaltz** [ImageMagick/ImageMagick#5484] [Development]
I have a C# app which converts Tiff files to a PDF but the result comes out with a black background and white text. The app calls a dll using Magick.NET-Q16-AnyCPU (v4.0.30319) and Magick.NET.Core (v2.0.50727). What confuses me is, the legacy version of this app works fine (white background with black text) but is using the same dll. 

What other factors could contribute to this behavior? The dll is .Net 4.6.1 as are both apps (legacy and current). I have copied the dll from the legacy app to the current but still have the problem. My next step is to recompile the dll but I don't know what to change.

Any thoughts? Thanks.

**Comment by dlemstra** [comment]
This sounds like an issue inside your own program? Is it possible to share a small sample that can be used to reproduce this problem?
